src/pyodide/internal/loadPackage.ts

```diff
@@ -129,15 +133,21 @@ async function loadPackagesImpl(
   const buffers = await Promise.all(loadPromises);
   for (const [requirement, reader] of buffers) {
     const [tarInfo, soFiles] = parseTarInfo(reader);
-    SITE_PACKAGES.addSmallBundle(tarInfo, soFiles, requirement);
+    const pkg = getPackageMetadata(requirement);
+    SITE_PACKAGES.addSmallBundle(
+      tarInfo,
+      soFiles,
+      requirement,
+      pkg.install_dir
+    );
   }
 
   console.log('Loaded ' + loading.join(', '));
 
   const tarFS = createTarFS(Module);
   const path = getSitePackagesPath(Module);
-  const info = SITE_PACKAGES.rootInfo;
-  Module.FS.mount(tarFS, { info }, path);
+  Module.FS.mount(tarFS, { info: SITE_PACKAGES.rootInfo }, path);
+  Module.FS.mount(tarFS, { info: SITE_PACKAGES.dynlibTarFs }, DYNLIB_PATH);
```

Comment:
Maybe `SITE_PACKAGES` is misnamed now that it includes both `SITE_PACKAGES` and the system lib directory.

Reply:
Done.